### DISCUSSION TITLE
Fan control requires app to run in windows store?

### DISCUSSION THREAD

**Discussion by JoshBatt1** [Rem0o/FanControl.Releases#1306] [Q&A]
When installing fan control and running the exe for the first time on a new pc a window pops up that says an app is required to run it. It automatically searches in the windows store for whatever app it needs but it doesn't find it.  What app is needed?

**Comment by Rem0o** [comment]
What? What setup? I think you may be mistaking an other software.

**Reply by JoshBatt1** [reply]
Sorry I meant when trying to run the .exe it says that an app is required to run it. Then it tries searching for it in the Microsoft store.

**Reply by Impact123** [reply]
You might want to attach a screenshot or something.

**Reply by Rem0o** [reply]
Oh you probably downloaded the net7_0 version, which requires the net framework 7.0, which doesn't come stock with Windows.  Get the 4.8 zip version instead if you don't want to deal with that.

**Reply by JoshBatt1** [reply]
Okay cool thank you, I’ll look into it. It’s on my friends pc so might be awhile.

**Reply by JoshBatt1** [reply]
So we tried installing the 4.8 version and it still doesnt work. This is what pops up after trying to run the .exe

![Capture](https://user-images.githubusercontent.com/96084869/203120488-5aab72f4-60da-4d55-9c6c-d06a9a661417.PNG)


> Oh you probably downloaded the net7_0 version, which requires the net framework 7.0, which doesn't come stock with Windows. Get the 4.8 zip version instead if you don't want to deal with that.

**Reply by JoshBatt1** [reply]
Nvm its fine now, had to right click on the fan.exe and go to properties and then click the unblock button. It now works
